packages/store/addon/-private/system/model/internal-model.ts

```diff
@@ -122,7 +122,6 @@ export default class InternalModel {
   __recordData: RecordData | null;
   _isDestroyed: boolean;
   isError: boolean;
-  _pendingRecordArrayManagerFlush: boolean;
```

Comment:
this state was only used in record-array-manager, so put it there with a `Set` of pending identifiers.